### PR TITLE
Branch ui countdown

### PR DESCRIPTION
**Pull request by cwenling**
To actually display the list of events and kind of display the first 2 in kind of prominent display. More work to be done. #77

### CODE REVIEW

File: src/main/java/seedu/tr4cker/ui/countdown/CountdownSecondEventBadge.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+public class CountdownSecondEventBadge extends UiPart<Region> {
+
+    private static final String FXML = "CountdownEventBadge.fxml";
+    private final Logger logger = LogsCenter.getLogger(seedu.tr4cker.ui.countdown.CountdownSecondEventBadge.class);
+
+    private final Event event;
+
+    @javafx.fxml.FXML
+    private Circle circle;
+    @FXML
```
Comment (hanweic53) [inline src/main/java/seedu/tr4cker/ui/countdown/CountdownSecondEventBadge.java:27]: add a line spacing after each field perhaps?

**Review by ethan-l-m-e — APPROVED** [review]
LGTM

**Review by hanweic53 — COMMENTED** [review]
Looks good, just a few nits to fix.

**Review by hanweic53 — APPROVED** [review]
Looks good to merge.

**Pull request merged**